test/functional/terminal/tui_spec.lua

```diff
@@ -2,9 +2,10 @@
 -- as a simple way to send keys and assert screen state.
 local helpers = require('test.functional.helpers')(after_each)
 local thelpers = require('test.functional.terminal.helpers')
-local feed = thelpers.feed_data
+local feed_data = thelpers.feed_data
```

Comment:
Overloading the usual functions like this is very confusing. Use feed_data explicitly to hint that the keys are sent to the child nvim instance.